backend/fetch_email.py

```diff
@@ -96,9 +87,3 @@ def handle_singlepart_message(self, message, result_text):
       result_text.append(body)
     elif content_type == "text/html":
       self.fetched_dict["html"] = body
-
-if __name__ == "__main__":
-  fetcher = EmailFetcher()
-  fetcher.fetch_account_emails(config.username, config.password)
```

Comment:
@Changhun-0305 Did you want me to leave this for testing purposes?

Reply:
Yeah if it's not a big deal, I think it would be make it easier to test.

Reply:
Fixed!